frontend/src/components/maps/FindMorePropertiesButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button, OverlayTrigger, Popover } from 'react-bootstrap';
+import { BsXSquareFill } from 'react-icons/bs';
+import { FaSign } from 'react-icons/fa';
+import styled from 'styled-components';
+import FindMorePropertiesForm from 'components/SearchBar/FindMorePropertiesForm';
+import { useMediaQuery } from 'react-responsive';
+
+const ButtonContent = styled.div`
+  display: flex;
+  .p {
+    font-size: 16px;
+  }
+`;
+
+/** popover that is triggered on button click to display form contents */
+const StyledPopover = styled(Popover)`
+  max-width: 100%;
+`;
+
+/** close button displayed in top right of popover title */
+const CloseButton = styled(BsXSquareFill)`
+  margin-left: 256px;
+  cursor: pointer;
+`;
+
+/** icon for sign used within component, styled blue */
+const TitleForSaleSign = styled(FaSign)`
+  fill: #003366;
+  margin-right: 5px;
+`;
+
+/** this component contains the trigger for additional filter options off the base filter */
+export const FindMorePropertiesButton: React.FC<any> = () => {
+  const smallScreen = useMediaQuery({ maxWidth: 1800 });
+  const TitleContent = () => (
+    <div style={{ display: 'flex' }}>
+      <TitleForSaleSign size={42} />
+      <h3 style={{ color: '#003366' }}>Find available surplus properties</h3>
+      <CloseButton onClick={() => document.body.click()} />
+    </div>
+  );
+
+  /** this provides a way to create a form with tooltip like behaviour in the overlay trigger */
+  const popover = (
```

Comment:
Having these functions like this will result in them being created each time the component is run.  Ideally you would `useCallback` or `useMemo`.   Or you would extract as a separate component.